templates/registration/logged_out.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}

```

Comment:
I guess it's better to show login page immediately.

Reply:
yep but i implemented this on the next task))